client/my-sites/plans-comparison/plans-comparison.tsx

```diff
@@ -453,6 +456,8 @@ export const PlansComparison: React.FunctionComponent< Props > = ( {
 	const prices = usePlanPrices( plans );
 	const translate = useTranslate();
 
+	const noAdsMonthlyCost = useSelector( getProductsList )?.[ WPCOM_FEATURES_NO_ADVERTS ]?.cost / 12;
```

Comment:
We have a [selector](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/products-list/selectors/get-product-cost.js) for product cost already, but why not turn this into a selector of its own? Something like `getProductMonthlyCost` makes sense. I guess also making `getProductCost` smarter may not be a bad idea (pass a 'period' or 'monthly' boolean to it).